internal/fs/inode/file.go

```diff
@@ -642,6 +646,27 @@ func (f *FileInode) flushUsingBufferedWriteHandler() error {
 	return nil
 }
 
+// SyncPendingBufferedWrites flushed any pending writes on the bwh to GCS.
+// It is a no-op when bwh is nil.
+//
+// LOCKS_REQUIRED((f.mu)
```

Comment:
Nit: sorry for the typo
```suggestion
// SyncPendingBufferedWrites flushes any pending writes on the bwh to GCS.
// It is a no-op when bwh is nil.
//
// LOCKS_REQUIRED(f.mu)
```